### PR TITLE
chore: removing mentions of plugin

### PR DESCRIPTION
**Pull request by sarahetter**
### Summary

This PR removes reference to "plugin" within this repo. The remaining references are to the deployment previews within our github actions (cypress), Use of `[[plugins]]` within `netlify.toml`, and import of the `NetlifyPlugin` from `@netlify/build`

### CODE REVIEW

File: docs/release-notes/v4.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Essential Next.js plugin v4 release notes
+# Next Runtime v4 release notes
```
Comment (ascorbic) [inline docs/release-notes/v4.md:1]: These are referring to old versions, so should keep the old name

File: demos/next-export/README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-This is an example of a site that uses `next export`. It only uses the plugin for cache handling and sets
+This is an example of a site that uses `next export`. It only uses the next runtime for cache handling and sets
```
Comment (ericapisani) [inline demos/next-export/README.md:1]: For consistency with the rest of the changes.

```suggestion
This is an example of a site that uses `next export`. It only uses the Next Runtime for cache handling and sets
```

File: docs/middleware.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ Next 12 introduces a new feature called [Middleware](https://nextjs.org/docs/mid
 a request has finished processing. Middleware can be used to modify the request or replace the response. For example, it
 can change headers, rewrite the request path, or return a different response entirely.
 
-Next.js Middleware can run either in an edge function (highly recommended for version 12.2+ as ISR will not work otherwise) or at the origin. On Netlify, middleware runs at the origin as
-part of the normal Next.js server.
+Next.js Middleware can run either in an edge function (highly recommended for version 12.2+ as ISR will not work
+otherwise) or at the origin. On Netlify, middleware runs at the origin as part of the normal Next.js server.
```
Comment (ericapisani) [inline docs/middleware.md:8]: Should this potentially be the following?

```suggestion
otherwise) or at the origin. On Netlify, Middleware runs at the origin as part of the normal Next.js server.
```

I'm unsure here if we're talking about middleware the feature (Next.js Middleware) or the general middleware concept/functionality that applies to any framework

File: docs/middleware.md [hunk 4]
```diff
@@ -43,14 +45,14 @@ export async function middleware(req: NextRequest) {
...
+that any pages that match middleware routes are served from the origin rather than the CDN.
 
 There is a bug in Next.js `<=12.0.3` that causes a proxy loop if you try to rewrite to a URL with a host other than
 localhost. This bug is fixed in version `12.0.4`, so if you are using middleware you should upgrade to that version or
 later.
 
 If you have an issue with Next.js middleware on Netlify while it is beta, particularly if the issue cannot be reproduced
 when running locally, then please add a comment to
-[the Next plugin beta discussion](https://ntl.fyi/next-beta-feedback).
+[the Next runtime beta discussion](https://ntl.fyi/next-beta-feedback).
```
Comment (ericapisani) [inline docs/middleware.md:58]: Should the 'r' be capitalized here?

```suggestion
[the Next Runtime beta discussion](https://ntl.fyi/next-beta-feedback).
```

File: docs/redirects-rewrites.md [hunk 5]
```diff
@@ -1,30 +1,58 @@
...
+
+Every site on Netlify supports [redirects and rewrites](https://docs.netlify.com/routing/redirects/), which are defined
+in a `_redirects` file or `netlify.toml`, and sites that use the Next Runtime are no exceptions. However there are some
+caveats to bear in mind when using them. The Next Runtime generates several rewrites of its own, which are used to map
+paths from the site to different Netlify functions which handle SSR, preview mode and images, as well as assets in
```
Comment (ericapisani) [inline docs/redirects-rewrites.md:13]: I think the 'f' is supposed to be capitalized, but might want to get a quick confirmation from @KyleBlankRollins 

```suggestion
paths from the site to different Netlify Functions which handle SSR, preview mode and images, as well as assets in
```

**Review by ericapisani — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,41 +1,41 @@
...
 </p>
 
 ## What's new in this version
 
 Version 4 is a complete rewrite of the Essential Next.js plugin. For full details of everything that's new, check out
-[the v4 release notes](https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/release-notes/v4.md)
+[the v4 release notes](https://github.com/netlify/next-runtime/blob/main/docs/release-notes/v4.md)
 
-## Installing the plugin
+## Installing the Next.js Runtime
```
Comment (KyleBlankRollins) [inline README.md:19]: [pebble] Because most folks will rely on on the automatic flow, we should rename this section to mention "manual". This makes it clearer that it's an optional, alternative path.

```suggestion
## Manually installing the Next.js Runtime
```

Comment (KyleBlankRollins) [inline README.md:19]: [sand] We may want to consider moving this section further down the page. Again, as most folks rely on the automatic flow, this section will be noise to them. They're more likely to be interested in other content in the readme.

File: README.md [hunk 7]
```diff
@@ -1,41 +1,41 @@
...
-package = "@netlify/plugin-nextjs"
+package = "@netlify/next-runtime"
 ```
 
 ## Deploying
 
-If you build on Netlify, this plugin will work with no additional configuration. However if you are building and
-deploying locally using the Netlify CLI, you must deploy using `netlify deploy --build`. Running the build and deploy
-commands separately will not work, because the plugin will not generate the required configuration.
+If you build on Netlify, this Next.js Runtime will work with no additional configuration. However if you are building
```
Comment (KyleBlankRollins) [inline README.md:25]: [sand] Minor edit to improve new wording flow.
```suggestion
If you build on Netlify, the Next.js Runtime will work with no additional configuration. However if you are building
```

File: README.md [hunk 8]
```diff
@@ -93,16 +95,16 @@ If you are using Nx, then you will need to point `publish` to the folder inside
 
 ## Incremental Static Regeneration (ISR)
 
-The Essential Next.js plugin now fully supports ISR on Netlify. For more details see
-[the ISR docs](https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/isr.md).
+The Next.js Runtime now fully supports ISR on Netlify. For more details see
```
Comment (KyleBlankRollins) [inline README.md:87]: [dust] Unrelated edit to remove time-based wording.
```suggestion
The Next.js Runtime fully supports ISR on Netlify. For more details see
```

File: docs/release-notes/v4.md (hunk 1 above)
Comment (KyleBlankRollins) [inline docs/release-notes/v4.md:1]: [pebble] I think we should leave this page and its content as Essential Next.js plugin. When v4 happened, that's what it was named. But the updated URLs are necessary to not break links. 😄

**Review by KyleBlankRollins — COMMENTED** [review]
Nice work, Sarah! I have some minor suggestions - nothing big. 🚀

**Review by ericapisani — APPROVED** [review]
